pkg/controller/storagecluster/noobaa_system_reconciler_test.go

```diff
@@ -98,7 +98,7 @@ func TestEnsureNooBaaSystem(t *testing.T) {
 
 	for _, c := range cases {
 		reconciler := getReconciler(t, &v1alpha1.NooBaa{})
-		reconciler.client.Create(context.TODO(), &cephCluster)//nolint //ignoring err check as causes failure
+		reconciler.client.Create(context.TODO(), &cephCluster) //nolint //ignoring err check as causes failure
```

Comment:
Does this change need to be part of this PR?

Reply:
Its linting issue. i cannot help it, it seems..... otherwise CI would fail

Reply:
Move this to it's own commit.

Reply:
I can't see how CI would fail on this - it would not have been passing on older PRs then. Unless you are using a different version of the linter locally.